_lessons/intro-to-jinja.html

```diff
@@ -195,6 +206,24 @@ <h1 style="font-size: 300%;" class="remark-inline-code">
 ```
 foo
 ```
+---
```

Comment:
this command / output was cut off the bottom of the slide, so I added a new slide for the second command / output